docs/EN_US/ECLStandardLibraryReference/SLR-Mods/DfuPlusExec.xml

```diff
@@ -23,36 +23,42 @@
         <row>
           <entry><emphasis>commandline</emphasis></entry>
 
-          <entry>A null-terminated string containing the DFUPlus.exe command
-          line to execute. The valid arguments are documented in the Client
-          Tools manual, in the section describing the DfuPlus.exe
+          <entry>A null-terminated string containing the DFUPlus command line
+          to execute. The valid arguments are documented in the Client Tools
+          manual, in the section describing the Command Line DFU
           program.</entry>
         </row>
       </tbody>
     </tgroup>
   </informaltable>
 
   <para>The <emphasis role="bold">DfuPlusExec</emphasis> action executes the
-  specified <emphasis>commandline</emphasis> just as the DfuPLus.exe program
-  would do. This simply allows you to have all the functionality of the
-  DfuPLus.exe program available within your ECL code.</para>
+  specified <emphasis>commandline</emphasis> just as the DfuPlus executable
+  program would. This allows you to have all the functionality of DfuPlus
+  available within your ECL code.</para>
+
+  <para>Unless you need to access a foreign instance of the platform, the
+  server= parameter for DfuPlus should be omitted, which then defaults to the
+  value contained in the environment's configuration. In a containerized
+  system, this is the value set in <emphasis
+  role="bold">global.defaultEsp</emphasis>. In a bare-metal system, this is
```

Comment:
@JamesDeFabia I'd actually rather they didn't set defaultEsp unless for some reason the system couldn't find the "default default eclservices" on it's own.  This should just work as long as they are running an internal eclservices as they should be.